src/app/core/config.py

```diff
@@ -48,6 +48,7 @@
 
 ELASTIC_INDEX = "workspace"
 ELASTIC_MAX_SIZE = 10000
+ELASTIC_TOTAL_SIZE = 1_000_000
```

Comment:
some comment about what this does would be nice at the config.py level :tongue:

Reply:
I added a short description, please review.